### DISCUSSION TITLE
Submit button has been removed in the View mode of Applications. Would there an Impact

### DISCUSSION THREAD

**Discussion by neethacjohn-aot** [AOT-Technologies/forms-flow-ai#1290] [Q&A]
The Submit button in the View mode of applications has been removed from the following pages

1. Tasks Page. For an unclaimed task, Submit button is not visible.When the task is claimed, Submit button becomes visible
![image](https://user-images.githubusercontent.com/69133582/223932242-13df0e18-ce85-4959-849f-110054f97b73.png)
![image](https://user-images.githubusercontent.com/69133582/223932349-728c1862-b05a-406b-930d-c8d4d7b39a9a.png)

2. View option in Application
![image](https://user-images.githubusercontent.com/69133582/223934084-568525af-af8b-458e-b184-d6933cfe4707.png)
![image](https://user-images.githubusercontent.com/69133582/223934209-a0aea769-3f4d-48e1-9219-cde2bd4f3cbf.png)

3. View submission option in Application
![image](https://user-images.githubusercontent.com/69133582/223934324-81cdba45-8aaf-4847-ae3e-402ce9040a65.png)

4. Form in Drafts
![image](https://user-images.githubusercontent.com/69133582/223934523-77ca1952-6ed3-4cdf-88ee-d8706064ebca.png)

5. View Submission in Tasks
![image](https://user-images.githubusercontent.com/69133582/223933651-e52f9211-32a5-4b0d-ba21-bbb12520e3f2.png)

Would there be an impact on the removal of the submit button in View Mode?

**Comment by shuhaib-aot** [comment]
It is not problematic to remove the submit button while in view mode, as it is not necessary to display it in that context. Formio offers two options for this purpose:

1. By setting the readOnly: true option, the form will be displayed in a disabled mode with the submit button.

2. The viewAsHtml: true option can be used to render the form in HTML mode, also without a submit button.
